### ISSUE TITLE
Give series their own OL ID

### ISSUE THREAD

**Issue by BrittanyBunk** [internetarchive/openlibrary#2848]
There are multiple reasons for this, but 1) they don't exist, 2) there are multiple works/volumes across series with no current way to connect them all, 3) differentiates series from catalogs - as there are multiple series in a catalog and the catalog numbers could go with the publishers books, 4) prevents excessive listing in work/edition descriptions, 5) helps with the organization of the site into the categories it needs from publisher -> translation

**Comment by Freso** [comment]
As a number of other databases have dedicated Series entities, it might be useful to look at some of them for how it could exist within Open Library. The external identifiers suggested in https://github.com/internetarchive/openlibrary/issues/9102 (and its comments) could be a good place to look at how they're implemented and/or presented in other systems.

**Comment by lokesh** [comment]
@cdrini completed?

**Comment by cdrini** [comment]
Yep completed! Currently limited to super librarians for editing/creating, but anyone can see them.
